package.json

```diff
@@ -26,14 +26,13 @@
     "gulp-minify-css": "^0.3.8",
     "gulp-print": "^1.1.0",
     "gulp-rename": "~1.2.0",
-    "gulp-ruby-sass": "1.1.0",
+    "gulp-sass": "^2.0.4",
```

Comment:
Can you also remove the installation of Ruby sass from the Dockerfile and make sure it builds OK?